test/Dialect/LLVMIR/rocdl.mlir

```diff
@@ -0,0 +1,35 @@
+// RUN: mlir-translate -mlir-to-rocdlir %s | FileCheck %s
```

Comment:
This folder should test the ops parsing, the verifiers (if any), and custom printing. This test does not seem in the right place, it seems like it is achieving the same thing as: https://github.com/tensorflow/mlir/blob/master/test/Target/nvvmir.mlir

Reply:
(Actually this is related to what you called-out, I noticed in parallel)